doc/api_methods/AddSubscriber.md

```diff
@@ -0,0 +1,44 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array addSubscriber(array $subsriber [, array $list_ids = [], array $options = []])`
+
+This method allows to create a subscriber, add him into lists and handles confirmation email and admin notification email sending and welcome email scheduling.
```

Comment:
> him

What if it is not him, but her? We should use "them" here and in all instances in this file.